mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/ui/game/ChessBoardTest.kt

```diff
@@ -80,6 +80,27 @@ class ChessBoardTest {
     }
   }
 
+  @Test
+  fun draggingPawnAround_withDisabledBoard_movesNothing() = runTest {
+    val state = SinglePieceSnapshotChessBoardState()
+    val strings =
+        rule.setContentWithLocalizedStrings {
+          ChessBoard(state, Modifier.size(160.dp).testTag("board"), enabled = false)
+        }
+    rule.onNodeWithTag("board").performTouchInput {
+      down(Offset(10.dp.toPx(), 10.dp.toPx()))
+      moveBy(Offset(0.dp.toPx(), 40.dp.toPx()))
+      up()
+    }
+    val bounds =
+        rule.onNodeWithContentDescription(
+                strings.boardPieceContentDescription(
+                    strings.boardColorWhite, strings.boardPiecePawn),
+            )
+            .getBoundsInRoot()
+    assertThat(DpOffset(10.dp, 10.dp) in bounds).isTrue()
+  }
+
   @Test
   fun draggingPawnAround_whileBoardIsSuccessful_dropsOnRightTarget() = runTest {
```

Comment:
I think this might be a more representative test name
```suggestion
  fun draggingPawnAround_whileBoardIsEnabled_dropsOnRightTarget() = runTest {
```